From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Mon, 5 Oct 2020 23:17:55 +0200
Subject: [ARCHEOLOGY] Switched RockPro64 to mainline u-boot (#2218)

> X-Git-Archeology: > recovered message: > * Switched RockPro64 to mainline u-boot
> X-Git-Archeology: > recovered message: > * Enabled rockusb and ums for RockPro64 in mainline u-boot
> X-Git-Archeology: > recovered message: > * Switched RockPro64 u-boot to TPL/SPL/BLOB scenario
> X-Git-Archeology: > recovered message: > * Enabled advanced recovery button support for RockPro64
> X-Git-Archeology: > recovered message: > * Disabled SDR104 mode for RockPro64 in legacy
> X-Git-Archeology: - Revision f2db96252530d2f5585755b518e7fd020b5f0392: https://github.com/armbian/build/commit/f2db96252530d2f5585755b518e7fd020b5f0392
> X-Git-Archeology:   Date: Mon, 05 Oct 2020 23:17:55 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched RockPro64 to mainline u-boot (#2218)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 3699d1bb68e29f1745ec214a37120afb0753a682: https://github.com/armbian/build/commit/3699d1bb68e29f1745ec214a37120afb0753a682
> X-Git-Archeology:   Date: Sat, 17 Oct 2020 23:39:29 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Configurable rockchip64 advanced u-boot recovery (#2255)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2a3c216d4107ddf30abadc6ba3c5707c1fa59ff2: https://github.com/armbian/build/commit/2a3c216d4107ddf30abadc6ba3c5707c1fa59ff2
> X-Git-Archeology:   Date: Mon, 04 Jan 2021 01:02:20 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched rockchip64/rk3399 to u-boot v2020.10 (#2512)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 86abac1fd4e169712a44d245cf7adef4ee17c420: https://github.com/armbian/build/commit/86abac1fd4e169712a44d245cf7adef4ee17c420
> X-Git-Archeology:   Date: Tue, 26 Oct 2021 22:14:41 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Cleanup u-boot configurations for rockchip64 derivatives (#3150)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 configs/rockpro64-rk3399_defconfig | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/configs/rockpro64-rk3399_defconfig b/configs/rockpro64-rk3399_defconfig
index d5e98a4f73..3f24728f20 100644
--- a/configs/rockpro64-rk3399_defconfig
+++ b/configs/rockpro64-rk3399_defconfig
@@ -28,6 +28,11 @@ CONFIG_CMD_GPT=y
 CONFIG_CMD_MMC=y
 CONFIG_CMD_PCI=y
 CONFIG_CMD_USB=y
+CONFIG_LED=y
+CONFIG_LED_GPIO=y
+CONFIG_CMD_LED=y
+CONFIG_CMD_ROCKUSB=y
+CONFIG_CMD_USB_MASS_STORAGE=y
 # CONFIG_CMD_SETEXPR is not set
 CONFIG_CMD_TIME=y
 CONFIG_SPL_OF_CONTROL=y
@@ -74,6 +79,10 @@ CONFIG_USB_OHCI_GENERIC=y
 CONFIG_USB_DWC3=y
 CONFIG_USB_DWC3_GENERIC=y
 CONFIG_USB_KEYBOARD=y
+CONFIG_USB_GADGET=y
+CONFIG_USB_FUNCTION_ROCKUSB=y
+CONFIG_ROCKCHIP_ADVANCED_RECOVERY=y
+CONFIG_ROCKCHIP_ADVANCED_RECOVERY_LED="work"
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
 CONFIG_USB_ETHER_ASIX88179=y
-- 
Armbian

